### PR TITLE
added php gd extension

### PR DESCRIPTION
**Pull request by rayzor65**
Fix was due to an issue of writing tests involving dompdf and phpunit

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -30,9 +30,11 @@ RUN apt-get install -y curl && \
     unzip \
     vim \
     libmcrypt-dev \
-    libxml2-dev
+    libxml2-dev \
+    libpng12-dev
 
-RUN docker-php-ext-install mcrypt pdo_pgsql mbstring pdo_mysql sockets opcache soap
+RUN docker-php-ext-configure gd --with-png-dir=/usr --with-jpeg-dir=/usr
```
Comment (zakhenry) [inline Dockerfile:36]: Does it work to configure before install?

Comment (alackmann) [inline Dockerfile:36]: Pretty sure it needs to be before the install - as it sets the defaults for that process

Comment (zakhenry) [inline Dockerfile:36]: Ak okay

Comment (rayzor65) [inline Dockerfile:36]: Yeah here's an example I saw, always configure first
https://hub.docker.com/r/tommylau/php/~/dockerfile/

**Pull request merged**